command/agent/config.go

```diff
@@ -516,6 +516,10 @@ type ServerConfig struct {
 	ExtraKeysHCL []string `hcl:",unusedKeys" json:"-"`
 
 	Search *Search `hcl:"search"`
+
+	// DeploymentRateLimit is in queries per second and is used by the
+	// DeploymentWatcher to throttle the amount of simultaneously deployments
+	DeploymentRateLimit float64
```

Comment:
I think this will get it parsed from the agent hcl. Also I think we should add at least the word `Query` so people don't assume this limits the number of concurrent deploys ... it's much more subtle than that for better or worse. The comment looks good though. :+1: 

```suggestion
	DeploymentQueryRateLimit float64 `hcl:"deploy_query_rate_limt"`
```